api/api-rules/violation_exceptions.list

```diff
@@ -516,6 +517,7 @@ API rule violation: names_match,k8s.io/api/core/v1,RBDVolumeSource,RBDPool
 API rule violation: names_match,k8s.io/api/core/v1,RBDVolumeSource,RadosUser
 API rule violation: names_match,k8s.io/api/core/v1,VolumeSource,CephFS
 API rule violation: names_match,k8s.io/api/core/v1,VolumeSource,StorageOS
+API rule violation: names_match,k8s.io/api/policy/v1,PodDisruptionBudgetStatus,PodDisruptionsAllowed
```

Comment:
this is the json tag mismatch violation... fixing the field name will fix this as well. unless you want to do custom conversions (pro-tip, you don't), you should change the field name in the internal and v1beta1 types as well